packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -4080,47 +4314,203 @@ curl -v -X POST \
+  "status": "SUCCESS",
+  "relayState": "/myapp/some/deep/link/i/want/to/return/to",
+  "sessionToken": "00t6IUQiVbWpMLgtmwSjMFzqykb5QcaBNtveiWlGeM",
+  "_embedded": {
+    "user": {
+      "id": "00ub0oNGTSWTBKOLGLNR",
+      "passwordChanged": "2015-09-08T20:14:45.000Z",
+      "profile": {
+        "login": "dade.murphy@example.com",
+        "firstName": "Dade",
+        "lastName": "Murphy",
+        "locale": "en_US",
+        "timeZone": "America/Los_Angeles"
+      }
+    }
+  }
+}
+```
+
+#### Verify Push Factor
+
+
+<ApiOperation method="post" url="/api/v1/authn/factors/${factorId}/verify" />
+
+Sends an asynchronous push notification (challenge) to the device for the user to approve or reject.  The `factorResult` for the transaction will have a result of `WAITING`, `SUCCESS`, `REJECTED`, or `TIMEOUT`.
+
+##### Request Parameters for Verify Push Factor
+
+
+| Parameter      | Description                                          | Param Type | DataType | Required |
+| -------------- | ---------------------------------------------------  | ---------- | -------- | -------- |
+| factorId       | `id` of factor returned from enrollment              | URL        | String   | TRUE     |
+| stateToken     | [state token](#state-token) for current transaction  | Body       | String   | TRUE     |
+| rememberDevice | user's decision to remember device                   | URL        | Boolean  | FALSE    |
+| autoPush       | user's decision to send push to device automatically | URL        | Boolean  | FALSE    |
+
+**Okta Verify Push Details Pertaining to Auto-Push**
+
+* You don't need to pass the `autoPush` flag to Okta unless you have a custom sign-in flow that does not use the Okta sign-in widget, but want Okta to keep track of this preference.  The custom sign-in flow must still handle the logic to actually send the Auto-Push, since this param only deals with the Auto-Push setting.
+* If you pass the `autoPush` query param when verifying an Okta Verify Push factor, Okta saves this value as the user's preference to have the push notification sent automatically if the verification is successful (the user presses "Approve" on their phone).
+* If there is already a saved Auto-Push preference, the successful verify call overrides the current preference if it is different from the value of `autoPush`.
+* This saved Auto-Push preference is always returned in the `/api/v1/authn/` response's `autoPushEnabled` field if the user is enrolled for the Okta Verify Push factor [example here](#response-example-for-factor-challenge-for-step-up-authentication-with-okta-session).  If the user's Auto-Push preference has not explicitly been set before, `autoPushEnabled` has a value of false.
+* The Auto-Push preference is stored in a cookie value and users that clear their cookies remove that preference.
+* Please note, the `autoPush` flag will have no effect when trying to verify a factor other than Okta Verify Push (factorId prefix = "opf").
```

Comment:
removing future tense

* Please note, the `autoPush` flag has no effect when trying to verify a factor other than Okta Verify Push (factorId prefix = "opf").